README.md

```diff
@@ -906,6 +907,129 @@ nvim_lsp#setup("rls", {config})
     settings = {}
 ```
 
+## solargraph
```

Comment:
this should go in `docs.description` i think? README.md is updated by a GH Action.

Reply:
I executed `sh ./scripts/docgen.sh`.
(https://github.com/neovim/nvim-lsp/blob/ac3df47ffd418b0f3aec49612d9862364820ed8d/CONTRIBUTING.md)

So, if we change the format, we should change https://github.com/neovim/nvim-lsp/blob/0d30787a47b4f4d04d5a08fe6eadf309c8e055b7/scripts/docgen.lua .